libs/eavmlib/src/network_fsm.erl

```diff
@@ -1,27 +1,7 @@
 %
```

Comment:
I'd be okay if we removed this module.  It's been deprecated for a long time.  But maybe not this PR..